reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServer.java

```diff
@@ -169,15 +170,18 @@ public final HttpServer accessLog(boolean enable) {
 	 *           .route(r -> r.get("/hello",
 	 *                   (req, res) -> res.header(CONTENT_TYPE, TEXT_PLAIN)
 	 *                                    .sendString(Mono.just("Hello World!"))))
-	 *           .accessLog(true, AccessLogFactory.create(
+	 *           .accessLog(AccessLogFactory.create(
```

Comment:
The example should stay with `boolean`

`.accessLog(true, AccessLogFactory.create(`

Reply:
nice catch, thanks